File: app/src/main/java/com/taoszu/configurer/CStudent.java
package com.taoszu.configurer;

import android.util.Log;

import com.taoszu.configurer.annotation.Worker;

@Worker(key = "C", module = "student",baseClass = BaseStudent.class)
public class CStudent implements BaseStudent {

  @Override
  public void printName() {
    Log.e("Student", "CStudent");
  }

}

File: app/src/main/java/com/taoszu/configurer/CStudent.kt
package com.taoszu.configurer

import android.util.Log
import com.taoszu.configurer.annotation.Worker

@Worker(key = ["B"], module = "LibStudent", baseClass = BaseStudentClass::class)
class CStudent : BaseStudentClass() {

  override fun printName() {
    Log.e("Student", "ATeacher")
  }

}